### PR TITLE
Feature/implement diff

### PR DESCRIPTION
**Pull request by oezelenes**

### CODE REVIEW

File: src/main/java/de/retest/recheck/cli/subcommands/Diff.java [hunk 1]
```diff
@@ -22,8 +24,15 @@
 
 	@Override
 	public void run() {
-		// TODO Implement.
-		logger.info( "Not yet implemented!" );
+		try {
+			logger.info( "\n{}", ReplayResultLoader.load( testReport ) );
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/cli/subcommands/Diff.java]: This uses `ReplayResult#toString()` but didn't we say we want to stick to those printers for now?

File: src/test/java/de/retest/recheck/cli/subcommands/DiffIT.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.SystemOutRule;
+import org.junit.rules.TemporaryFolder;
+
+import de.retest.recheck.cli.util.ReportCreator;
+import picocli.CommandLine;
+
+public class DiffIT {
+
+	private final TemporaryFolder temp = new TemporaryFolder();
```
Comment (beatngu13) [inline src/test/java/de/retest/recheck/cli/subcommands/DiffIT.java]: Isn't here `@Rule` missing? Then you can probably also omit the `setUp()` method.

File: src/main/java/de/retest/recheck/cli/ReplayResultLoader.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.retest.recheck.LoadRecheckIgnoreUtil;
+import de.retest.recheck.configuration.Configuration;
+import de.retest.recheck.persistence.Persistence;
+import de.retest.recheck.persistence.bin.KryoPersistence;
+import de.retest.recheck.report.ReplayResult;
+
+public class ReplayResultLoader {
```
Comment (beatngu13) [inline src/main/java/de/retest/recheck/cli/ReplayResultLoader.java]: Good change! 👍

File: src/test/java/de/retest/recheck/cli/util/ReportCreator.java [hunk 4]
```diff
@@ -1,21 +1,98 @@
...
+import de.retest.recheck.ui.descriptors.IdentifyingAttributes;
+import de.retest.recheck.ui.descriptors.PathAttribute;
+import de.retest.recheck.ui.descriptors.RootElement;
+import de.retest.recheck.ui.descriptors.StringAttribute;
+import de.retest.recheck.ui.descriptors.SutState;
+import de.retest.recheck.ui.descriptors.TextAttribute;
+import de.retest.recheck.ui.diff.RootElementDifference;
+import de.retest.recheck.ui.diff.RootElementDifferenceFinder;
 
 public class ReportCreator {
```
Comment (modulo11) [inline src/test/java/de/retest/recheck/cli/util/ReportCreator.java]: We should consider moving this helper to recheck and generalize all path related stuff.

**Pull request merged**